pkg/reconciler/dependencybuild/buildrecipeyaml.go

```diff
@@ -366,87 +380,118 @@ func createPipelineSpec(log logr.Logger, tool string, commitTime int64, jbsConfi
 					},
 				},
 				{
-					Name: "GIT_IDENTITY",
+					Name: "RECIPE_IMAGE",
 					Value: tektonpipeline.ParamValue{
 						Type:      tektonpipeline.ParamTypeString,
-						StringVal: jbsConfig.Spec.GitSourceArchive.Identity,
+						StringVal: recipe.Image,
 					},
 				},
 				{
-					Name: "GIT_URL",
+					Name: "BUILD_TOOL",
 					Value: tektonpipeline.ParamValue{
 						Type:      tektonpipeline.ParamTypeString,
-						StringVal: jbsConfig.Spec.GitSourceArchive.URL,
+						StringVal: tool,
 					},
 				},
 				{
-					Name: "GIT_SSL_VERIFICATION",
+					Name: "BUILD_TOOL_VERSION",
 					Value: tektonpipeline.ParamValue{
 						Type:      tektonpipeline.ParamTypeString,
-						StringVal: strconv.FormatBool(jbsConfig.Spec.GitSourceArchive.DisableSSLVerification),
+						StringVal: recipe.ToolVersion,
 					},
 				},
 				{
-					Name: "GIT_REUSE_REPOSITORY",
+					Name: "JAVA_VERSION",
 					Value: tektonpipeline.ParamValue{
 						Type:      tektonpipeline.ParamTypeString,
-						StringVal: strconv.FormatBool(db.Annotations[artifactbuild.DependencyScmAnnotation] == "true"),
+						StringVal: recipe.JavaVersion,
 					},
 				},
 				{
-					Name: "SCM_URL",
+					Name: "BUILD_SCRIPT",
 					Value: tektonpipeline.ParamValue{
 						Type:      tektonpipeline.ParamTypeString,
-						StringVal: db.Spec.ScmInfo.SCMURL,
+						StringVal: konfluxScript,
 					},
 				},
+				// This will break various parts of JBS as the PNC tooling does NOT support disabling plugins on
+				// a per build basis.
+				//
+				//{
+				//	Name: "BUILD_PLUGINS",
+				//	Value: tektonpipeline.ParamValue{
+				//		Type:      tektonpipeline.ParamTypeString,
+				//		StringVal: strings.Join(recipe.DisabledPlugins, ","),
+				//	},
+				//},
 				{
-					Name: "SCM_HASH",
+					Name: "JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE",
 					Value: tektonpipeline.ParamValue{
 						Type:      tektonpipeline.ParamTypeString,
-						StringVal: db.Spec.ScmInfo.CommitHash,
+						StringVal: "quay.io/redhat-user-workloads/konflux-jbs-pnc-tenant/konflux-tooling:latest",
```

Comment:
Bit of hard-coding here - normally `JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE` is used but now prebuild is targeted at PNC and uses the separate https://github.com/project-ncl/konflux-tooling project we need to split them out and hardcode this.